File: recovered/src/com/lumiyaviewer/lumiya/render/shaders/RiggedMeshProgram.java
package com.lumiyaviewer.lumiya.render.shaders;

import android.opengl.GLES20;

/* loaded from: classes.dex */
public class RiggedMeshProgram extends PrimProgram {
    public int uBindShapeMatrix;
    public int uJointVectors;
    public int vJoint;
    public int vWeight;

    public RiggedMeshProgram(boolean z) {
        super(Shader.RiggedMeshVertexShader, z ? Shader.PrimOpaqueFragmentShader : Shader.PrimFragmentShader);
    }

    @Override // com.lumiyaviewer.lumiya.render.shaders.PrimProgram, com.lumiyaviewer.lumiya.render.shaders.BasicPrimProgram, com.lumiyaviewer.lumiya.render.shaders.ShaderProgram
    protected void bindVariables() {
        super.bindVariables();
        this.uBindShapeMatrix = GLES20.glGetUniformLocation(this.handle, "uBindShapeMatrix");
        this.uJointVectors = GLES20.glGetUniformLocation(this.handle, "uJointVectors");
        this.vWeight = GLES20.glGetAttribLocation(this.handle, "vWeight");
        this.vJoint = GLES20.glGetAttribLocation(this.handle, "vJoint");
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/render/shaders/RiggedMeshProgram.kt
package com.lumiyaviewer.lumiya.render.shaders

import android.opengl.GLES20

class RiggedMeshProgram(opaque: Boolean) : PrimProgram(
    Shader.RiggedMeshVertexShader,
    if (opaque) Shader.PrimOpaqueFragmentShader else Shader.PrimFragmentShader
) {
    @JvmField var uBindShapeMatrix = 0
    @JvmField var uJointVectors = 0
    @JvmField var vJoint = 0
    @JvmField var vWeight = 0

    override fun bindVariables() {
        super.bindVariables()
        uBindShapeMatrix = GLES20.glGetUniformLocation(handle, "uBindShapeMatrix")
        uJointVectors = GLES20.glGetUniformLocation(handle, "uJointVectors")
        vWeight = GLES20.glGetAttribLocation(handle, "vWeight")
        vJoint = GLES20.glGetAttribLocation(handle, "vJoint")
    }
}
